src/Tribe/Tickets_View.php

```diff
@@ -163,8 +195,10 @@ public function update_tickets() {
 	 * @return void
 	 */
 	public function authorization_redirect() {
+		global $wp_query;
+
 		// When it's not our query we don't care
-		if ( ! tribe_is_event_query() ) {
+		if ( empty( $wp_query->tribe_is_event_query ) ) {
```

Comment:
Hrm. Well then. Perhaps we should add a function to tribe-common that TEC can hook to so that event-tickets doesn't need to be aware of TEC stuff? Maybe a function called `tribe_is_modified_query()` that has a `tribe_is_modified_query` filter. Then core can use `tribe_is_event_query()` to impact that filtered value in some way?